src/VbaCompiler/README.md

```diff
@@ -0,0 +1,37 @@
+Library `VbaCompiler` compiles Visual Basic source code and Ribbon customizations to Microsoft Office macro enabled files.
+
+### Usage
+
+Sample usage to generate Excel workbook file with macro
+from the source code files `Module.vb` and `MyClass.vb`:
+
+```csharp
+using DocumentFormat.OpenXml.Packaging;
+
+var compiler = new VbaCompiler();
+compiler.ProjectId = Guid.NewGuid();
+compiler.ProjectName = "My Macro Project";
+compiler.CompanyName = "ACME";
+compiler.AddModule("Module.vb");
+compiler.AddClass("MyClass.vb");
+
+// generate the vbaProject.bin file
+var vbaProjectPath = compiler.CompileVbaProject("obj", "vbaProject.bin");
+
+// generate Excel macro file
+var macroFile = compiler.CompileExcelMacroFile("bin", "MyMacro.xlsm", vbaProjectPath, SpreadsheetDocumentType.MacroEnabledWorkbook);
```

Comment:
I think the `macroFilePath` would be more fitting name for this example here, so it's clear, that the path to the file is returned, not the file directly - as is for the vbaProjectPath above.